example-logger.py

```diff
@@ -0,0 +1,8 @@
+import statsd
```

Comment:
I think this file isn't needed right?

Reply:
Well, for debugging maybe, but it indeed a bit outside the scope of this project, I'll remove it